cheatsheets/Transport_Layer_Security_Cheat_Sheet.md

```diff
@@ -136,19 +136,17 @@ For internal applications, an internal CA can be used. This means that the FQDN
 
 Certification Authority Authorization (CAA) DNS records can be used to define which CAs are permitted to issue certificates for a domain. The records contains a list of CAs, and any CA who is not included in that list should refuse to issue a certificate for the domain. This can help to prevent an attacker from obtaining unauthorized certificates for a domain through a less-reputable CA. Where it is applied to all subdomains, it can also be useful from an administrative perspective by limiting which CAs administrators or developers are able to use, and by preventing them from obtaining unauthorized wildcard certificates.
 
-### Always Provide All Needed Certificates
+### Consider the Certificate’s Validation Type
 
-In order to validate the authenticity of a certificate, the user's browser must examine the certificate that was used to sign it and compare it to the list of CAs trusted by their system. In many cases the certificate is not directly signed by a root CA, but is instead signed by an intermediate CA, which is in turn signed by the root CA.
+Certificates come in types of validation. Validation is the process the Certificate Authority uses to make sure you are allowed to have the certificate. This is authorization. The base validation is called Domain Validated (DV). All publicly issued certificates must be domain validated. This process involves practical proof of control of the name or endpoint requested in the certificate. This usually involves a challenge and response in DNS, to an official email address, or to the endpoint that will get the certificate.
```

Comment:
It's really the CA/Browser that defines the validations, right? So perhaps more accurate to say that the certificates include information about who they are issued to and how was it verified. On the web, the required processes are defined by browser vendors and CAs in collaboration.

Reply:
I think the validation is defined in the TLS and PKIX related RFCs, although the CAB Forum may influence IETF in that regard. @markgamache probably has them all memorized. :rofl: